dashboard/lib/services/i18n/curriculum_sync_utils/sync_in.rb

```diff
@@ -21,7 +21,10 @@ def self.serialize
 
             # prepare data
             data = ScriptCrowdinSerializer.new(script).as_json.compact
-            data.delete(:crowdin_key) # don't need this for top-level data
+            # The JSON object will have the script's crowdin_key as the top level key, but we don't
+            # need that, so we will discard the crowdin_key and set data to be the object it is
+            # pointing to.
+            data = data.first[1] unless data.first.nil?
```

Comment:
not sure I understand how this is different than deleting the key

Reply:
Before this change, at the top level of the JSON it looked liked this:
```json
{
  "crowdin_key": "/s/express-2021/lessons/1",
  "lessons": {
    "/s/express-2021/lessons/1/levels/2": {
      "short_instructions": "In this lesson you will be..."
    }
  }
}
```
The previous developers didn't think the top level `crowdin_key` was needed, so a simple call to `object.delete('crowdin_key')` was all that was needed. Now that all objects are serialized our custome way, the `crowdin_key` is moved outside the object. Therefore the structure is slightly different so we will need to delete it a different way.
```json
{
  "/s/express-2021/lessons/1": {
    "lessons": {
      "/s/express-2021/lessons/1/levels/2": {
        "short_instructions": "In this lesson you will be..."
      }
    }
  }
}
```
`data = data.first[1] unless data.first.nil?` makes it look like this:
```json
{
  "lessons": {
    "/s/express-2021/lessons/1/levels/2": {
      "short_instructions": "In this lesson you will be..."
    }
  }
}
```
Note that the key "/s/express-2021/lessons/1" was removed and the Hash it pointed to replaced it.